src/main/java/de/tum/in/www1/artemis/service/compass/controller/ModelSelector.java

```diff
@@ -2,13 +2,16 @@
 
 import java.util.*;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
 
 import de.tum.in.www1.artemis.service.compass.umlmodel.classdiagram.UMLClassDiagram;
 
 public class ModelSelector {
 
     private static final int MAX_CANDIDATE_LIST_SIZE = 50;
 
+    private static final double EPSILON = 0.0000001;
```

Comment:
What is this used for? Could you shortly document both constants?

Reply:
done.